mnefun/_mnefun.py

```diff
@@ -32,13 +32,14 @@
     from mne import compute_raw_data_covariance as compute_raw_covariance
 from mne.preprocessing.ssp import compute_proj_ecg, compute_proj_eog
 from mne.preprocessing.maxfilter import fit_sphere_to_headshape
+from mne.preprocessing.maxwell import maxwell_filter
 from mne.minimum_norm import make_inverse_operator
 from mne.label import read_label
 from mne.epochs import combine_event_ids
-try:
-    from mne.chpi import _quat_to_rot, _rot_to_quat
-except ImportError:
-    from mne.io.chpi import _quat_to_rot, _rot_to_quat
+#try:
+#    from mne.chpi import _quat_to_rot, _rot_to_quat
+#except ImportError:
+#    from mne.io.chpi import _quat_to_rot, _rot_to_quat
```

Comment:
there will be conflicts on these lines, they've been changed in `master`. I didn't do it with a PR but probably should have, sorry about that.